### PR TITLE
added Verify Phone Finder test and relevant pages

### PR DESCRIPTION
**Pull request by ashchavinska**

### CODE REVIEW

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java [hunk 1]
```diff
@@ -238,4 +238,43 @@ public void verifyBurgerMenuInHeader() {
...
+
+        List<ModelItem> listOfModelItems = phoneFinderResultPage.getListModelItem();
+        Assert.assertFalse(listOfModelItems.isEmpty(), "Search result is empty");
+        for (ModelItem item : listOfModelItems) {
+            Assert.assertTrue(item.readModel().toLowerCase().contains(elementToSearch.get("searchKeyword").toLowerCase()), "Search result is not as required");
+        }
+
+        Assert.assertTrue(phoneFinderResultPage.isTextInBottomPresent(), "Text in bottom isn't present");
+
+        PhoneFinderPage refiningPhoneFinderPage = phoneFinderResultPage.clickClickHereBtn();
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: you created PhoneFinderPage above and you can use it

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java [hunk 2]
```diff
@@ -238,4 +238,43 @@ public void verifyBurgerMenuInHeader() {
...
+        Assert.assertEquals(refiningPhoneFinderPage.getSelectedBrand().toLowerCase(), elementToSearch.get("searchKeyword").toLowerCase(), "ghj" );
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: add isPageOpened before this assert

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java (hunk 1 above)
Comment (ashchavinska) [reply src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: done

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java (hunk 2 above)
Comment (ashchavinska) [reply src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: done

**Comment by vrudyk** [comment]
ok

**Pull request merged**